src/api/public/apidocs-new/OBS-v2.10.50.yaml

```
@@ -255,7 +255,8 @@ paths:
     $ref: 'paths/source_orderkiwirepos.yaml'
 
   # Sources - Projects
-
+  /source:
+    $ref: 'paths/source_projects.yaml'
```

```suggestion
    $ref: 'paths/source.yaml'
```
Let's keep the convention that every path file has the name of its path and arguments.